.github/workflows/macos.yml

```diff
@@ -28,6 +28,7 @@ jobs:
 
       - name: Install Qt ${{ matrix.qt-version-major }} (homebrew)
         run: |
+          export HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK=1
```

Comment:
Nitpick: Move this env var out of the build commands and add a `TODO` pointing to the upstream issue:

```yaml
- name: Install Qt ${{ matrix.qt-version-major }} (homebrew)
  env:
    # TODO: Remove once https://github.com/actions/setup-python/issues/577 gets fixed.
    HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK: 1
  run: |
    brew update
```